wodles/aws/buckets_s3/aws_bucket.py

```diff
@@ -528,6 +528,10 @@ def iter_events(self, event_list, log_key, aws_account_id):
                         f'+++ The "{self.discard_regex.pattern}" regex found a match in the "{self.discard_field}" '
                         f'field. The event will be skipped.', 2)
                     continue
+                else:
+                    aws_tools.debug(
+                        f'+++ The "{self.discard_regex.pattern}" regex did not match any value in the '
+                        f'"{self.discard_field}" field. The event will be processed.', 3)
```

Comment:
To be consistent with the message printed when the regex finds a match, I'd suggest to adjust the message

```suggestion
                        f'+++ The "{self.discard_regex.pattern}" regex did not find a match in the '
                        f'"{self.discard_field}" field. The event will be processed.', 3)
```